config/jobs/kubernetes-csi/gen-jobs.sh

```diff
@@ -316,6 +316,16 @@ snapshotter_version() {
     fi
 }
 
+use_bazel() {
+    local kubernetes="$1"
+
+    # Kubernetes 1.21 removed support for building with Bazel.
+    if version_gt "$kubernetes" "1.21"; then
+        echo "false"
+    else
+        echo "true"
+    fi
+}
```

Comment:
We could avoid adding this function and simple pass `CSI_PROW_USE_BAZEL=false` to all jobs. Then jobs which currently build images for Kubernetes 1.20 would also stop using Bazel although at that time, using Bazel was still the recommended way of building. I felt it was safer to follow recommendations for each release, hence this version check.

Reply:
this is pretty much what we're doing with rest of CI for now, and especially kind jobs have this env based approach.

though because we carry some extra complexity in kind for building w/ bazel and are re-working image builds I may drop this between now and the next kind release and either:
- pin kubernetes release branch CI to kind v0.10.0
- go ahead and switch the release branch CI over to make for kind jobs at least (lower impact the further back you go anyhow)